src/server/multi_command_squasher.cc

```diff
@@ -207,8 +211,24 @@ bool MultiCommandSquasher::ExecuteSquashed() {
     tx->PrepareSquashedMultiHop(base_cid_, cb);
     tx->ScheduleSingleHop([this](auto* tx, auto* es) { return SquashedHopCb(tx, es); });
   } else {
+#if 1
+    fb2::BlockingCounter bc(num_shards);
+    DVLOG(1) << "Squashing " << num_shards << " " << tx->DebugId();
+
+    auto cb = [this, tx, bc]() mutable {
+      this->SquashedHopCb(tx, EngineShard::tlocal());
+      bc->Dec();
+    };
+
+    for (unsigned i = 0; i < sharded_.size(); ++i) {
+      if (!sharded_[i].cmds.empty())
+        shard_set->AddL2(i, cb);
+    }
+    bc->Wait();
+#else
```

Comment:
I kept it to keep debugging why there is a memory blow up with `RunBlockingInParallel`

Reply:
Yeah but you still have to rebuild it, so copy paste replace would've done it 😛